src/rules/indentation/index.js

```diff
@@ -0,0 +1,133 @@
+import _ from "lodash"
```

Comment:
I guess you can only use `import { repeat } from "lodash"` or just use the modularized `lodash.repeat` maybe https://www.npmjs.com/package/lodash.repeat ?